oi/type_graph/TypeIdentifier.cpp

```diff
@@ -20,6 +20,15 @@
 
 namespace type_graph {
 
+// TODO:
+// - read these from a TOML file
```

Comment:
can you create an issue for putting these in the config or creating container `.toml`s for them? I'd lean towards the config for simplicity but either works

Reply:
Agreed that just putting these in the config would be simpler - no need for a file for each